client/my-sites/plans-single-products/index.jsx

```diff
@@ -15,20 +14,16 @@ import Product from './product';
  */
 import './style.scss';
 
-const PlansSingleProducts = ( { translate } ) => {
+const PlansSingleProducts = () => {
 	const displayBackup = isEnabled( 'plans/jetpack-backup' );
 	const displayScan = isEnabled( 'plans/jetpack-scan' );
 
-	if ( ! displayBackup && ! displayScan ) {
-		return null;
-	}
-
 	return (
 		<div className="plans-single-products">
-			{ displayScan && <Product heading={ translate( 'Jetpack Scan' ) } /> }
-			{ displayBackup && <Product heading={ translate( 'Jetpack Backup' ) } /> }
+			{ displayScan && <Product productSlug="jetpack-scan" /> }
```

Comment:
To me it makes more sense to that we pass in the properties in here. vs build them up in the `Product` component. That will make the product component more generic and more reusable.

Reply:
I was taking this approach into consideration, too. I think, though, that even if we promote the properties to the parent component, the `Product` component still won't be *that* reusable.

`Product` is quite tightly coupled with `PlansSingleProducts` since they are sharing the CSS class name base and stylesheet itself.

Nevertheless, I'm going to move the properties up to the parent component.